front/src/config/i18n/en.json

```diff
@@ -283,6 +321,11 @@
       "title": "MQTT",
       "deviceTab": "Devices",
       "setupTab": "Setup",
+      "status": {
+        "notConfigured": "MQTT service is not configured, please go to ",
```

Comment:
Please go to?

Reply:
To be able to use `preact-router Link`, the link is outside the i18n : 

![NotConfigured](https://user-images.githubusercontent.com/1839717/67507634-9e69c980-f68f-11e9-94d4-724196793bf8.PNG)

Reply:
My bad!